flink-libraries/flink-table/src/main/java/org/apache/calcite/sql/fun/SqlStdOperatorTable.java

```diff
@@ -1039,7 +1039,7 @@ public boolean argumentMustBeScalar(int ordinal) {
 		new SqlCollectionTableOperator("TABLE", SqlModality.RELATION);
 
 	public static final SqlOverlapsOperator OVERLAPS =
```

Comment:
I think we can remove this file for now and rely on Calcite 1.13 implementation. We don't need it anymore.